.github/CODEOWNERS

```diff
@@ -1,40 +1,54 @@
-# The following volunteers have self-identified as subject matter experts
-# or interested parties over a particular area of the php-src source code.
-# While requesting a review from someone does not obligate that person to
-# review a pull request, these reviewers might have valuable knowledge of
-# the problem area and could aid in deciding whether a pull request is ready
-# for merging.
+# The following volunteers have self-identified as subject matter experts or
+# interested parties in a particular area of the php-src source code. While
+# requesting a review from someone does not obligate that person to review a
+# pull request, these reviewers might have valuable knowledge of the problem
+# area and could aid in deciding whether a pull request is ready for merging.
```

Comment:
I'd leave that paragraph as is; in my opinion, the former formatting is slightly better readable.

Reply:
Changed back to the "free-style". Note, that there was also a slight grammatical/contextual fix. But ok. Let's have the initial text.

Reply:
Ah, I've overlooked the "over a particular area" to "in a particular area" change. That one was good. To feel to put it in again.

Reply:
Yeah, I'm not really sure about that either. Don't we have a (section of a) document which could be linked? https://wiki.php.net/vcs/gitworkflow#bugfix_workflow_for_core_developers seems not to be focused enough, so might not actually be read.

Reply:
There is CONTRIBUTING.md file which has this described a bit: https://github.com/php/php-src/blob/master/CONTRIBUTING.md#pull-requests 
I've added link to it instead.